services/QuillLMS/config/application.rb

```diff
@@ -73,5 +73,9 @@ class Application < Rails::Application
     config.public_file_server.headers = {
       'Access-Control-Allow-Origin' => '*'
     }
+
+    config.active_record.encryption.primary_key = ENV['ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY']
+    config.active_record.encryption.deterministic_key = ENV['ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY']
+    config.active_record.encryption.key_derivation_salt = ENV['ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT']
```

Comment:
A comment might be helpful here, answering:

- do we ever need to change the salt?
- how do we recover if the salt value is deleted or misconfigured?